packages/orchestrator-ui-components/src/rtk/api.ts

```diff
@@ -85,6 +85,24 @@ export const orchestratorApi = createApi({
                 const graphqlFn = graphqlRequestBaseQuery({
                     url: customApi ? customApi.apiBaseUrl : graphqlEndpointCore,
                     prepareHeaders,
+                    customErrors: (error) => {
+                        const { name, message, stack, response } = error;
+                        if (
+                            response &&
+                            response.errors &&
+                            response.errors.length > 0
```

Comment:
Can be replaced with `response?.errors?.length > 0`

Reply:
Fixd